project_task_date_shift/views/project_task.xml

```diff
@@ -0,0 +1,48 @@
+<odoo>
+  <!-- List View -->
+  <record id="project_task_date_shift_view_tree" model="ir.ui.view">
+    <field name="name">project.task.date.shift.tree</field>
+    <field name="model">project.task</field>
+    <field name="inherit_id" ref="project.view_task_tree2"/>
+    <field eval="2" name="priority"/>
+    <field name="arch" type="xml">
+      <field name="user_id" position="after">
+        <field name="date_start"/>
+        <field name="date_end"/>
+      </field>
+    </field>
+  </record>
+
+  <!-- Form View -->
+  <record id="project_task_date_shift_view_form" model="ir.ui.view">
+    <field name="name">project.task.date.shift.form</field>
+    <field name="model">project.task</field>
+    <field name="inherit_id" ref="project.view_task_form2"/>
+    <field eval="2" name="priority"/>
+    <field name="arch" type="xml">
+      <data>
+        <xpath expr="//field[@name='date_deadline']" position = "before">
+          <field name="date_start"/>
+          <field name="date_end"/>
+        </xpath>
+      </data>
+    </field>
+  </record>
+
+  <!-- Gantt View -->

```

Comment:
Gantt View is a Timeline view?

Reply:
I changed it to <!-- Gantt/Timeline View --> to be more clear.